cc.config.json

```diff
@@ -1,12 +1,45 @@
 {
-    "platforms": {
-        "NATIVE": {
-            "moduleOverrides": {
+    "features": {
+        "base": { "modules": [ "base" ] },
+        "gfx-webgl": { "modules": [ "gfx-webgl" ] },
+        "gfx-webgl2": { "modules": [ "gfx-webgl2" ] },
+        "ui": { "modules": [ "ui" ] },
+        "particle": { "modules": [ "particle" ] },
+        "physics-cannon": { "modules": [ "physics-cannon", "physics-framework" ] },
+        "physics-ammo": { "modules": [ "physics-ammo", "physics-framework", "wait-for-ammo-instantiation" ] },
+        "physics-builtin": { "modules": [ "physics-builtin", "physics-framework" ] },
+        "physics-2d-box2d": { "modules": [ "physics-2d-box2d", "physics-2d-framework" ] },
+        "physics-2d-builtin": { "modules": [ "physics-2d-builtin", "physics-2d-framework" ] },
+        "intersection-2d": { "modules": [ "intersection-2d" ] },
+        "audio": { "modules": [ "audio" ] },
+        "video": { "modules": [ "video" ] },
+        "terrain": { "modules": [ "terrain" ] },
+        "webview": { "modules": [ "webview" ] },
+        "tween": { "modules": [ "tween" ] },
+        "tiled-map": { "modules": [ "tiled-map" ] },
+        "spine": { "modules": [ "spine" ] },
+        "dragonbones": { "modules": [ "dragonbones" ] }
+    },
+    "index": {
+        "modules": {
+            "spine": { "ns": "sp" }
+        }
+    },
+    "moduleOverrides": [
+        {
+            "test": "context.mode === 'BUILD'",
+            "overrides": {
+                "cocos/core/data/deserialize-dynamic.ts": "cocos/core/data/deserialize-dynamic-empty.ts"
+            }
+        },
+        {
+            "test": "context.platform === 'NATIVE'",
```

Comment:
@minggo

Reply:
It is ok to me if it can replace the .ts files correctly.